src/course-search/FacetListBase.jsx

```diff
@@ -84,6 +99,12 @@ FacetListBase.propTypes = {
   isCheckedField: PropTypes.string,
   items: PropTypes.arrayOf(PropTypes.shape()).isRequired,
   title: PropTypes.string.isRequired,
+  typeahead: PropTypes.shape({
```

Comment:
nit/recommendation: I wonder if `typeaheadOptions` might be a bit more understandable? My first thought at the name `typeahead` is that it was a boolean variable, when it's actually an object.

Reply:
(unless this is one of those props coming from Algolia!)